spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageProtocolResolver.java

```diff
@@ -64,6 +64,6 @@ public Resource resolve(String location, ResourceLoader resourceLoader) {
 		if (location.startsWith("gs://")) {
 			return new GoogleStorageResource(this.storage, location);
 		}
-		return this.delegate.getResource(location);
+		return null;
```

Comment:
Sorry, has missed your comment here.


Good choice is to add some comment in the next commit any way.
Remember? We squash everything before merging to the `master`.
So, the history of commits in this PR doesn't matter.
Only the history in the master is important.
And that way without comment in the commit it would be hard to understand what was the point of braking change on this line even if we know that you are guilty in the change.

Again, it's IMHO how I feel things should go with the hope that my ideas would be helpful.
I may adapt to any schema if that works for the whole team.